src/mms_client/types/base.py

```diff
@@ -76,16 +76,16 @@ class ProcessingStatistics(BaseXmlModel):
     timestamp_xml: Optional[DateTime] = attr(default=None, name="XmlTimeStamp")
 
 
-class ResponseCommon(BaseXmlModel):
+class ResponseCommon(BaseXmlModel, search_mode="unordered"):
```

Comment:
I have never seen this syntax before... supplying an argument to a class definition?

Reply:
It weirded me out as well, but it's part of the `pydantic_xml` standard apparently. Basically, it's saying that I don't care what order the elements appear in, when deserializing from XML.

Reply:
Thanks! I hate it! Approved.

Reply:
Maybe this would more clear for us?
```suggestion
class ResponseCommon(BaseXmlModel, search_mode="unordered"):
    def __init__(self, search_mode="unordered"):
        super().__init__(search_mode=search_mode)
```

Reply:
If that works, I would definitely prefer that.